src/overlays/actors/ovl_Door_Spiral/z_door_spiral.c

```diff
@@ -31,30 +40,223 @@ const ActorInit Door_Spiral_InitVars = {
     (ActorFunc)DoorSpiral_Init,
     (ActorFunc)DoorSpiral_Destroy,
     (ActorFunc)DoorSpiral_Update,
-    (ActorFunc)DoorSpiral_Draw
+    (ActorFunc)DoorSpiral_Draw,
+};
+
+typedef struct {
+    /* 0x0 */ Gfx* dLists[2]; // represents a staircase up or down
```

Comment:
Comment is good, could you also copy this comment onto `/* 0x146 */ u8 direction;`